ext/intl/formatter/formatter.stub.php

```diff
@@ -92,6 +92,16 @@ class NumberFormatter
      * @cvalue UNUM_ROUND_UP
      */
     public const ROUND_UP = UNKNOWN;
+    /**
+     * @var int
+     * @cvalue UNUM_ROUND_DOWN
+     */
+    public const ROUND_TOWARD_ZERO = UNKNOWN;
+    /**
+     * @var int
+     * @cvalue UNUM_ROUND_UP
+     */
+    public const ROUND_AWAY_FROM_ZERO = UNKNOWN;
```

Comment:
I think this should be done in a separate follow up PR

Reply:
I have included this change in the same RFC. Do you think I should create separate PR for this, or it's fine to keep it in this?

Reply:
I left this in the same PR as it was accepted in the same RFC.